.github/workflows/ci.yml

```
@@ -0,0 +1,16 @@
+name: Release
```

Кажется не очень удачное название.
```suggestion
name: Lint
```

Файл я бы тоже переименовал.